### ISSUE TITLE
This is just to point out that I added code to defect 13 comments.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#14]
```
This is just to point out that I added code to defect 13 comments.
```

Original issue reported on code.google.com by `ibunto...@gmail.com` on 21 Jul 2007 at 6:53

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 1.5
```

Original comment by `ibunto...@gmail.com` on 24 Sep 2007 at 10:55
- Added labels: **Milestone-Release1.5**

**Comment by GoogleCodeExporter** [comment]
```
This was fixed and verified earlier
```

Original comment by `ibunto...@gmail.com` on 15 Aug 2008 at 9:09
- Changed state: **Verified**
